src/app-logic/l10n.js

```diff
@@ -14,7 +15,10 @@ export const DEFAULT_LOCALE = 'en-US';
  * Returns the locale and the ftl string grouped as an Array.
  */
 export async function fetchMessages(locale: string): Promise<[string, string]> {
-  const response = await fetch(`/locales/${locale}/app.ftl`);
+  const response =
+    locale === DEFAULT_LOCALE
+      ? await fetch(engFtlFile)
```

Comment:
The goal is that we don't need to use `fetch` :-) Using webpack's ["raw loader"](https://v4.webpack.js.org/loaders/raw-loader/) (instead of the "file-loader"), using "import" we would have the full text in the variable (with the file-loader we have just the URL, as you've seen).

__However__ I think this doesn't work with Jest actually. This is a bit tricky: Jest (our test runner) doesn't use webpack so everything "special" we do through webpack configuration, we have to find a workaround with Jest as well.
I forgot about this with my idea, so I think my idea isn't practical after all.

I'll think of something else and write down a suggestion later today.

Reply:
I have a question here, here we are actually doing `fetch` for both the conditions right? Does the `import engFtlFile` effect in any way?

Reply:
Here we can revert the change :-) this was a bad suggestion from me!